include/aspect/material_model/interface.h

```diff
@@ -26,7 +26,17 @@
 #include <aspect/material_model/utilities.h>
 
 #include <deal.II/base/point.h>
+
+// Work around an incorrect instantiation in qprojector.h of deal.II 9.2.0,
```

Comment:
Very strictly speaking, it's not an instantiation that is in that file (which would be preceded by `template ...`) but a declaration of a specialization (preceded by `template <> ...`), but who cares. This is going to go away in a year or two anyway :-)